dc-cudami-editor/src/components/dialogs/ConfirmGeneratatedUrlAliasesDialog.jsx

```diff
@@ -1,47 +1,73 @@
-import groupBy from 'lodash/groupBy'
+import {useState} from 'react'
 import {useTranslation} from 'react-i18next'
 import {
   Button,
   ButtonGroup,
   FormGroup,
   Label,
+  ListGroup,
   Modal,
   ModalBody,
   ModalFooter,
   ModalHeader,
 } from 'reactstrap'
 
-import UrlAliases from '../UrlAliases'
+import {EditableUrlAlias, UrlAlias} from '../UrlAliases'
 
 const ConfirmGeneratatedUrlAliasesDialog = ({
   isOpen,
   generatedUrlAliases = {},
+  onChange,
   onConfirm,
   toggle,
 }) => {
+  const [editable, setEditable] = useState(false)
   const {t} = useTranslation()
   return (
-    <Modal isOpen={isOpen} size="lg" toggle={toggle}>
-      <ModalHeader toggle={toggle}>{t('generatedUrlAliases')}</ModalHeader>
+    <Modal isOpen={isOpen} size="lg">
+      <ModalHeader
+        cssModule={{
+          'modal-title': 'd-flex justify-content-between modal-title w-100',
+        }}
+      >
+        {t('generatedUrlAliases')}
+        {!editable && (
+          <Button color="primary" onClick={() => setEditable(true)} size="xs">
+            {t('edit')}
+          </Button>
+        )}
+      </ModalHeader>
       <ModalBody>
-        {Object.entries(generatedUrlAliases).map(([language, aliases]) => (
+        {Object.entries(generatedUrlAliases).map(([language, [alias]]) => (
```

Comment:
Hier wird doch nur der erste aus der Liste gezogen. Könnten in dieser Liste auch mehrere drin sein? (Vllt. beim Editieren?)

Reply:
Da können nie mehrere drin sein, beim Editieren existieren die generierten Links ja schon - bei Neuanlage wird pro Sprache einer erzeugt.

Reply:
Nope. Haben wir geklärt.